frontend/src/metabase/admin/permissions/selectors/data-permissions/tables.ts

```diff
@@ -79,21 +82,24 @@ const buildAccessPermission = (
       DATA_PERMISSION_OPTIONS.controlled,
       originalValue === DATA_PERMISSION_OPTIONS.noSelfServiceDeprecated.value &&
         DATA_PERMISSION_OPTIONS.noSelfServiceDeprecated,
+      ...PLUGIN_ADMIN_PERMISSIONS_TABLE_OPTIONS,
     ]),
     value,
   );
 
+  const isDisabled =
+    isAdmin ||
+    (!isAdmin &&
+      (options.length <= 1 ||
+        PLUGIN_ADVANCED_PERMISSIONS.isAccessPermissionDisabled(
+          value,
+          "tables",
+        )));
```

Comment:
```suggestion
  const isDisabled =
    isAdmin
    || options.length <= 1
    || PLUGIN_ADVANCED_PERMISSIONS.isAccessPermissionDisabled(value, "tables");
```

Reply:
I realize you just moved these lines of code from somewhere else, but they seem worth making a little clearer

Reply:
good catch!